Formula/f/firefly.rb

```diff
@@ -0,0 +1,31 @@
+class Firefly < Formula
+  desc "Command-line Interface (CLI) for Hyperledger FireFly"
```

Comment:
So what can people actually use it for? Because most of Homebrew/core is a CLI

Reply:
This CLI is designed for creating an environment, after which you can interact with it using various commands. However, the process of creating the environment requires an interactive prompt, which makes it challenging to automate in non-interactive tests. That said, I did add a test case that verifies if the engine is not running, which provides some validation of the tool's functionality.

Reply:
So like this?
```suggestion
  desc "Create environments with Hyperledger FireFly"
```

Reply:
@SMillerDev  Not really an environment more like a stack - let me explain

Hyperledger FireFly is a tool for interacting with blockchains and is composed of multiple micro services that need to talk to each other. The FireFly CLI provides a way to setup those micro services and a blockchain node in docker-compose for developers and users to quickly get started with the system. The CLI will create the docker compose file, create volumes, etc.. to be able to run the FireFly stack. 

Alongside, it also has a set of command to interact with the APIs provide by FireFly.

Reply:
I'm fine with any explanation, but currently most of the description is redundant with the default in Homebrew core and the other part is redundant with the formula name. I'd like it to actually describe something to users of Homebrew.
Block 1 describes what essentially every formula is, block 2 is the formula name:
```rb
desc "[Command-line Interface (CLI)] for Hyperledger [FireFly]"
```

Reply:
@SMillerDev It's a good point, I updated the description :)